src/governance.rst

```diff
@@ -0,0 +1,172 @@
+  after a previous votes failed to achieve Sam's result. Sam is
+  attempting to bully other members of core into agreeing, and is thus
+  violating the code of conduct.
+
+Voting items are labeled as either **standard** or **sensitive**.
+Standard items are ones where public record and discourse and record is
+preferable. Sensitive voting items are ones where the results of the
+vote should remain private to the voters after the vote has occurred.
+Sensitive votes should take place on
+`the Helios voting system <https://vote.heliosvoting.org/>`_ in order
+retain anonymity.
+
+The default voting period is 1 week (7 days). This may be modified at
+the time when a vote is called, but may never be less than 24 hrs.
+
+:Nominate new member of staged-recipes: The proposer must provide
+    a brief justification as to why the new member is desirable or needed.
+
+    * Sensitive
+    * 50% Majority to pass
+
+:Nominate new member of core: The proposer must provide
+    a sufficient justification as to why the nominee should be welcomed
+    into core. It is necessary that all nominees are current and active
+    members of staged-recipes. Nominees to core must have been members
+    of staged-recipes for at least 1 month (4 weeks) prior to this vote.
+
+    * Sensitive
+    * 66.7% Majority to pass
+
+:Sub-team Formation: Proposers must specify the name, role & responsibility,
+    members, and charter (dynamic or static) of any new sub-teams.
+
+    * Standard
+    * 50% Majority to pass
+
+:Sub-team Dissolution: Proposers must specify the name and justification
+    for why a sub-team should be dissolved.
+
+    * Standard
+    * 50% Majority to pass
+
+:Lock an Issue, Pull Request, Thread: Occasionally, discussions become
+    toxic and antithetical to the goal of fostering the conda-forge
+    community. Members of core have the right to call a vote to lock
+    the thread. Such a call must be legitimate and well justified.
+    This power may not be used to stop valid discussions between those
+    with differing opinions on the technical matter at hand.
+
+    * Standard
+    * 50% Majority to pass
+
+:Block a Contributor: In extreme cases, such as repeated harassment,
+    it may become necessary to block a user completely from participating
+    in all conda-forge activities. This should not be done lightly,
+    but it may be necessary to do so expediently. Shorter voting periods
+    (such as 24 hrs) are to be expected. The proposer of the block
+    must provide ample justification as to why this is needed.
+
+    * Sensitive
+    * 60% Majority to pass
+
+
+:Remove member of staged-recipes: The proposer must provide
+    a justification as to why the member of staged recipes
+    should be removed.
+
+    * Sensitive
+    * 66.7% Majority to pass
+
+:Remove member of core: The proposer must provide
+    an overwhelming justification as to why the member core
+    should be removed.
+
+    * Sensitive
+    * 75% Majority to pass
```

Comment:
Commenting here, but this impacts removals and blocks above as well.

It seems like warnings should be raised to the individual or individuals before we come to blocking/removing them. Suggesting this as the individual may not be aware there actions are not inline with the org's expectations (written or otherwise).

How do we do this while still being sensitive as well? For example, mentioning we might block people in a public sphere is likely to have a chilling effect on the community; so, what private communications are appropriate?

How we decide to warn an individual? Does that need a quorum? Potentially a smaller one or specific team dedicated to this function? What determines the make up of this team? How do other orgs handle this?

How many warnings are appropriate? Does this vary depending on the action and/or the member's standing in the org.

Are there forms of behavior that are so egregious that they may deserve fewer or no warnings?

Sorry for the list of questions. Just want to make sure we are giving this a reasonable level of thought.